### PR TITLE
Cap supported kubernetes version to <1.22 in chart

### PR DESCRIPTION
**Pull request by cmurphy**
Kubernetes 1.21 introduced an in-tree admission controller to mitigate
the CVE that this webhook is created for. This change prevents
installation of the chart on 1.22, giving users one release of overlap
to migrate from the webhook to the admission controller.

https://github.com/rancher/rancher/issues/33893

### CODE REVIEW

File: chart/Chart.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ description: |
 name: rancher-externalip-webhook
 version: %VERSION%
 appVersion: %APP_VERSION%
+kubeVersion: < 1.21.2
```
Comment (cbron) [inline chart/Chart.yaml]:
```suggestion
kubeVersion: < 1.21.2-0
```

Comment (cmurphy) [reply chart/Chart.yaml]: Actually meant `1.22.0` but was using 1.21 for testing :woman_facepalming:

Comment (cbron) [reply chart/Chart.yaml]: Oh, I was assuming 1.21.2 was when it was introduced upstream. I suppose that date doesn't matter.

File: chart/Chart.yaml [hunk 2]
```diff
@@ -11,6 +11,7 @@ description: |
 name: rancher-externalip-webhook
 version: %VERSION%
 appVersion: %APP_VERSION%
+kubeVersion: < 1.22.0
```
Comment (cbron) [inline chart/Chart.yaml:14]: still need 
```suggestion
kubeVersion: < 1.22.0-0
```

Comment (cbron) [reply chart/Chart.yaml:14]: Actually this doesn't matter, we don't do anything with kube alphas typically. I was thinking rancher version.

**Review by cbron — APPROVED** [review]

**Review by rawmind0 — APPROVED** [review]
LGTM

**Pull request merged**